redo/rules/build_style.py

```diff
@@ -85,6 +85,28 @@ def _build(self, redo_1, redo_2, redo_3):
             shell.run_command(flake_cmd)
             os.chdir(cwd)
 
+        # Run codespell check:
+        codespell_cwd = os.getcwd() + os.sep + "*.*"
+        codespell_ignore = os.environ["ADAMANT_DIR"] + os.sep + "redo" + os.sep + "codespell" + os.sep + "ignore_list.txt"
+        shell.run_command("codespell "
+                          + codespell_cwd
+                          + " -I "
+                          + codespell_ignore
+                          + " -S "
+                          + os.path.basename(style_log_file)
+                          + " 2>&1 | tee -a "
+                          + style_log_file
+                          + " 1>&2")
+        shell.run_command("codespell "
+                          + build_directory
+                          + " -I "
+                          + codespell_ignore
+                          + " -S "
+                          + os.path.basename(style_log_file)
+                          + " 2>&1 | tee -a "
+                          + style_log_file
+                          + " 1>&2")
```

Comment:
Maybe try to combine some of the code duplication here to make maintenance easier.